kafka/src/main/resources/reference.conf

```diff
@@ -93,6 +95,14 @@ hydra {
       key.deserializer = org.apache.kafka.common.serialization.StringDeserializer
       value.deserializer = io.confluent.kafka.serializers.KafkaAvroDeserializer
     }
+
+    //compacted streams config
+    parallelism = 3
+    eos-commit-interval = 100
```

Comment:
we aren't using a transactional sink, but eos-commit-interval is needed by CommiterSettings

Reply:
look at the parallelism to verify it's correct